src/api/lib/tasks/dev/requests.rake

```diff
@@ -224,5 +224,341 @@ namespace :dev do
+          end
+        elsif repository['arch']
+          proj_repo.repository_architectures.find_or_initialize_by(architecture: Architecture.find_by(name: repository['arch']))
+        end
+
+        proj_repo.save!
+        project.store(comment: "Added #{proj_repo.name} repository")
+      end
+    end
+    # rubocop:enable Metrics/CyclomaticComplexity
+    # rubocop:enable Metrics/PerceivedComplexity
+
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    def create_users_and_groups_from_meta(request_data:)
+      # create users
+      if request_data['person'].present?
+        [request_data['person']].flatten.each do |person|
+          find_user(person['userid']) if person['userid']
+        end
+      end
+
+      return if request_data['group'].nil?
+
+      if request_data['group'].is_a?(Array)
+        request_data['group'].each do |group|
+          next if Group.find_by(title: group['groupid'])
+
+          create(:group, title: group['groupid'])
+        end
+      else
+        group = Group.find_by(title: request_data['group']['groupid'])
+
+        create(:group, title: request_data['group']['groupid']) if group.nil?
+      end
+    end
+    # rubocop:enable Metrics/CyclomaticComplexity
+    # rubocop:enable Metrics/PerceivedComplexity
+
+    def find_user(login)
+      alias_for_login(login)
+      user = User.find_by_login(@users_list[login])
+      user ||= create(:confirmed_user, login: @users_list[login])
+
+      user
+    end
+
+    def alias_for_login(login = nil)
+      return if login.blank?
+
+      @users_list[login] ||= Faker::Alphanumeric.alpha(number: 10)
+    end
+
+    def find_group(title)
+      return if title.nil?
+
+      Group.find_or_create_by!(title: title)
+    end
+
+    def find_project(project_name)
+      return if project_name.nil?
+
+      Project.find_or_create_by(name: project_name)
+    end
+
+    def find_package(project_name, package_name)
+      return if project_name.nil? || package_name.nil?
+
+      package = Package.find_by_project_and_name(project_name, package_name)
+      return package if package
+
+      project = Project.find_or_create_by(name: project_name)
+      create(:package, name: package_name, project: project)
+    end
+
+    def clone_prj_configs(config:, comment:, project:)
+      project.config.save({ user: User.session!.login, comment: comment }, config)
+    end
+
+    def make_api_request(url:, params: {}, headers: { 'Content-Type' => 'application/xml' })
+      username = ''
+      password = ''
+
+      abort("#{'=' * 50}\nusername or password not present.") unless username.present? && password.present?
+
+      conn = Faraday.new(
+        url: url,
+        params: params,
+        headers: headers
+      )
+      conn.set_basic_auth(username, password)
+      request = conn.get
+
+      request.body
+    end
+
+    def base_api_url
+      'https://api.opensuse.org'
+    end
+
+    def print_message(message)
```

Comment:
You can remove this

Reply:
I think we should keep it because it works like a logger to inform about newly created requests. Any reason why you would like to drop this?

Reply:
I think we can find out the generated data in the UI already. Having it as console output is just noise, in my opinion

Reply:
After discussion we have decided to keep this